### PR TITLE
Bumping up the normal label's background colors

### PR DESCRIPTION
**Pull request by StefanIvanov**

### CODE REVIEW

**Comment by StefanIvanov** [comment]
All has been ran by Sassy and these are the improvements we make bumping all default labels' colors a bit

**Review by Ddungyov — APPROVED** [review]

**Comment by StefanIvanov** [comment]
I will wait until our next sync to обнародвам and then merge this :)

**Review by kckfm1 — APPROVED** [review]

**Pull request merged**
